### PR TITLE
``[Exiled::CustomRoles]`` ``[Exiled::CustomItems]`` QoL in List commands.

### PR DESCRIPTION
**Pull request by SrLicht**
## Description
**Describe the changes** 

Using the Custom Item and Custom Roles list commands will execute the registered list if there is no argument.

**What is the current behavior?** (You can also link to an open issue here)
Needless to write registered for a list command and Custom Roles does not say that the ``customrole list abilities`` command exists.

**What is the new behavior?** (if this is a feature change)

Typing the command without arguments will execute the registered command of the list command.

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:
Testing result:
### Custom item command
imagen

### Custom roles command
imagen

<br />

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [X] I have checked the project can be compiled
- [X] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [ ] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

### CODE REVIEW

File: EXILED/Exiled.CustomRoles/Commands/List/List.cs [hunk 1]
```diff
@@ -45,7 +45,14 @@ public override void LoadGeneratedCommands()
         /// <inheritdoc/>
         protected override bool ExecuteParent(ArraySegment<string> arguments, ICommandSender sender, out string response)
         {
-            response = "Invalid subcommand! Available: registered.";
+            if (arguments.IsEmpty() && TryGetCommand(Registered.Instance.Command, out ICommand command))
+            {
+                command.Execute(arguments, sender, out response);
+                response += $"\nTo view all abilities registered use command: {arguments.Array[0]} {arguments.Array[1]} abilities";
```
Comment (louis1706) [inline EXILED/Exiled.CustomRoles/Commands/List/List.cs:51]: use string.Join array

File: EXILED/Exiled.CustomItems/Commands/List/List.cs [hunk 2]
```diff
@@ -45,7 +45,14 @@ public override void LoadGeneratedCommands()
         /// <inheritdoc/>
         protected override bool ExecuteParent(ArraySegment<string> arguments, ICommandSender sender, out string response)
         {
-            response = $"Invalid subcommand! Available: registered, insideinventories";
+            if (arguments.IsEmpty() && TryGetCommand(Registered.Instance.Command, out ICommand command))
+            {
+                command.Execute(arguments, sender, out response);
+                response += $"\nTo view custom items in players' inventories, use the command: {arguments.Array[0]} {arguments.Array[1]} insideinventories";
```
Comment (louis1706) [inline EXILED/Exiled.CustomItems/Commands/List/List.cs:51]: Use string.Join

**Comment by louis1706** [comment]
That not how i meant it to be 

Replave Array[1] Array[0] by using string.join

**Comment by SrLicht** [comment]
> That not how i meant it to be
> 
> Replave Array[1] Array[0] by using string.join

XD OH man i need to sleep

**Review by louis1706 — APPROVED** [review]

**Pull request merged**